content/reference/current/plugins/jenkins/index.md

```diff
@@ -49,6 +49,19 @@ You can use the Pipeline Snippet Generator to help you use the Jenkins Plugin. C
 
 You can specify the id of an API Token stored in a secret text credential if you don't want to use the one configured globally. Choose one of the simulation in the drop-down, then click Generate Groovy. Copy and paste the result in your Pipeline script, eg:
 ```groovy
+// Declarative Pipeline Syntax
+pipeline {
+    agent any
+    stages {
+        stage("Gatling Enterprise simulation") {
+            steps {
+                gatlingFrontLineLauncherStep credentialId: '6737158c-0ff6-4033-91ad-6f3a811aab52', '00eacd1c-ef91-4076-ad57-99b4c6675a9e'
```

Comment:
**nit:** second parameters not named?

Reply:
**issue (not in this PR):** step name should not have "Step"

As we will have to rename it to remove "FrontLine", I propose:
 * `gatlingStartSimulation`
 * `gatlingLaunchSimulation`
 * `gatlingRunSimulation` (my preference if this step wait for the run to end)

Reply:
1. this is the documentation for the current live version.
2. what you suggest is a breaking change, not something we can do any time soon. We can introduce new commands though and deprecate the current ones. Please open a dedicated ticket, this is not this PR's scope.